src/platform/forms/save-in-progress/FormSignInModal.jsx

```diff
@@ -28,8 +28,9 @@ class FormSignInModal extends React.Component {
       action: this.handleSignIn,
       text: 'Sign in and start over',
     };
-    const { formConfig } = this.props;
-    const { appType } = formConfig?.customText?.appType || APP_TYPE_DEFAULT;
```

Comment:
From what I can tell, the real fix was to not destructure something that isn't meant to be destructured. Note that destructuring a string results in `undefined`.

Reply:
The fix should just be:

```js
const appType = formConfig?.customText?.appType || APP_TYPE_DEFAULT;
```

Reply:
Exactly.

Reply:
nit: and might as well use `??` instead of `||`

Reply:
Are there cases for having an empty string or other falsy value for `appType`? It seems it would result in the buggy error message from the screenshot above.

Reply:
`APP_TYPE_DEFAULT` should be used if any falsy values came in for `appType`.